src/Coverage.Report/Interfaces/ILcovConvertForwardingApp.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Coverage.Report.Interfaces
+{
+    using System.Collections.Generic;
+
+    public interface ILcovConvertForwardingApp
```

Comment:
Should we document the interfaces?

Reply:
What does the forwarding app do?

Reply:
> Should we document the interfaces?

Yes, Will do in this PR or next PR. I want to unblock the contributes ASAP.

>What does the forwarding app do?

Didn't think much about it. It is copied from [VSTestForwardingApp](https://github.com/Microsoft/vstest/blob/master/src/Microsoft.TestPlatform.Build/Tasks/VSTestForwardingApp.cs). I'm thinking Tasks classes just have responsibility to get parameter from target. The core logic should happen in forwarding app(or any better name we can give).